Gopkg.lock

```diff
@@ -922,7 +923,6 @@
     "github.com/openshift/library-go/pkg/crypto",
     "go.uber.org/zap",
     "k8s.io/api/apps/v1",
-    "k8s.io/api/autoscaling/v1",
```

Comment:
In case anyone is curious, https://github.com/openshift/cluster-ingress-operator/pull/157 dropped this import because we stopped using the scale client in the `TestClusterIngressScale` E2E test while we sort out an API issue that affects CRDs with the `/scale` subresource.